broadcaster/_backends/gcloud_pubsub.py

```diff
@@ -0,0 +1,112 @@
+import asyncio
+import typing
+import functools
+from urllib.parse import urlparse
+
+from google.cloud import pubsub_v1
+
+from .._base import Event
+from .base import BroadcastBackend
+
+
+class GCloudPubSubBackend(BroadcastBackend):    
```

Comment:
Watch out for trailing spaces.
```suggestion
class GCloudPubSubBackend(BroadcastBackend):
```
Otherwise I have nothing useful to add. :-)